monitor/main.go

```diff
@@ -1,36 +1,114 @@
 package monitor
 
 import (
+	"bytes"
 	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
 	"math/big"
+	"net/http"
 	"os"
 	"path/filepath"
+	"sync"
 	"time"
 
 	"github.com/celo-org/eksportisto/db"
 	"github.com/celo-org/eksportisto/metrics"
 	"github.com/celo-org/eksportisto/utils"
 	"github.com/celo-org/kliento/client"
+	"github.com/celo-org/kliento/client/debug"
 	"github.com/celo-org/kliento/contracts"
 	kliento_mon "github.com/celo-org/kliento/monitor"
 	"github.com/celo-org/kliento/registry"
 	"github.com/ethereum/go-ethereum"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/ethereum/go-ethereum/log"
 	"golang.org/x/sync/errgroup"
 )
 
 type Config struct {
-	NodeUri string
-	DataDir string
+	NodeUri                   string
+	DataDir                   string
+	SensitiveAccountsFilePath string
 }
 
 var EpochSize = uint64(17280)   // 17280 = 12 * 60 * 24
 var BlocksPerHour = uint64(720) // 720 = 12 * 60
 var TipGap = big.NewInt(50)
 
+func getSensitiveAccounts(filePath string) map[common.Address]string {
```

Comment:
I'm assuming the string is the callback/webhook url?
If so, maybe that needs to be represented in the name?

Reply:
`getSensitiveAccountWebhookUrls`?